### PR TITLE
Hot fix onlineRegistration issue

### PR DESCRIPTION
**Pull request by MarkWang2**
#### Decription
http://www.active.com/orangeville-on/water-sports/swimming-classes/cpr-recert-2030
The event can not be registration but the register now button shows.
#### Test Link
http://a3coreint15.dev.activenetwork.com/orangeville-on/water-sports/swimming-classes/cpr-recert-2030

### CODE REVIEW

File: spec/actv/event_spec.rb [hunk 1]
```diff
@@ -49,37 +49,66 @@ def format_date_in_utc date
...
   describe '#online_registration_available?' do
-    context 'when online_registration_available is true' do
-      before do
-        subject.legacy_data.stub(:onlineRegistration).and_return("true")
-        subject.stub(:registrationUrlAdr).and_return("something")
+    context "when legacy_data is present" do
+      context "when legacy_data field online_registration is string value 'true'" do
+        before do
+          subject.legacy_data.stub(:onlineRegistration).and_return("true")
+          subject.stub(:registrationUrlAdr).and_return("something")
```
Comment (marvin-min) [inline spec/actv/event_spec.rb]: we could move the duplicate `subject.stub(:registrationUrlAdr).and_return("something")` to the line under `context "when legacy_data is present" do`

**Comment by marvin-min** [comment]
Looks good to me.

File: spec/actv/event_spec.rb [hunk 2]
```diff
@@ -49,37 +49,50 @@ def format_date_in_utc date
...
-        subject.legacy_data.stub(:onlineRegistration).and_return("true")
-        subject.stub(:registrationUrlAdr).and_return("something")
-      end
+    context "when legacy_data is present" do
+      context "when registrationUrlAdr is present" do
+        before { subject.stub(:registrationUrlAdr).and_return("something") }
 
-      its(:online_registration_available?) { should be_true }
-    end
+        context "when legacy_data field online_registration is string value 'true'" do
```
Comment (helloaiway) [inline spec/actv/event_spec.rb]: The syntax  looks strange.
How about:
"when legacy_data field online_registration is string and the value is true"

**Comment by hlcfan** [comment]
Looks good now.

**Pull request merged**